core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -58,14 +59,16 @@ export class NavigationComponent {
 
   // whether user dashboard is enabled and accessible from the workspace
   public userSystemEnabled: boolean = environment.userSystemEnabled;
+  // flag to display a particular version in the current canvas
+  public displayParticularWorkflowVersion: boolean = false;
```

Comment:
I feel we have a flag in the service but another flag in the component. it is hard to manage two copies and they can easily out of sync to create false logic. can we unify them? for example, make this variable listen to the change event of the variable in service.

Reply:
the component is actually listening to the event(flag) from service and only echoing the change